src/DetailsView/components/issues-table.tsx

```diff
@@ -182,7 +182,11 @@ export class IssuesTable extends React.Component<IssuesTableProps> {
     }
 
     private renderSpinner(label: string): JSX.Element {
-        return <Spinner className="details-view-spinner" size={SpinnerSize.large} label={label} />;
+        return (
+            <div role="alert">
```

Comment:
It says [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) that:

> To maximize compatibility, some people recommend adding a redundant aria-live="assertive" when using this role. However, adding both aria-live and role="alert" causes double speaking issues in VoiceOver on iOS.

I'm not sure which is better, but `ScanningSpinner` has the redundant property, so we may want to be consistent

Reply:
Yeap, I tried using '`aria-live`' here same as on the electron case, but for some reason, NVDA and JAWS didn't produce the desire effect (I haven't test VoiceOver though). This reason is related to my answer on your other comment.

Reply:
Updated